Makefile

```
@@ -157,8 +157,13 @@ kind-deploy-controller: generate-operator-yaml install-resources deploy
 .PHONY: deploy-controller
 deploy-controller: kind-deploy-controller
 
+HOSTED ?= none
+
 .PHONY: kind-deploy-controller-dev
-kind-deploy-controller-dev: kind-deploy-controller build-images
+kind-deploy-controller-dev:
```

I removed `build-images ` because in grc-framework already has` build-images `step  https://github.com/stolostron/governance-policy-framework/blob/0903452ed5de2b5d99eba286f4a16ef1007ed064/.github/workflows/kind.yml#L160

---

Removing `build-images` is good, but I'm still curious whether swapping the `CONTROLLER_NAMESPACE` and `KIND_NAMESPACE` values letting the `kind-deploy-controller` target run here works. If so, it'd let us test deployment YAML changes.
```suggestion
kind-deploy-controller-dev: kind-deploy-controller
```

---

@dhaiducek  Should we create a separate ticket for `CONTROLLER_NAMESPACE and KIND_NAMESPACE`? looks like only config-controller-repo has this

---

We don't need kind-deploy-controller step because addon-repo(framework install this repo) already added the controller.

---

Right, but if the deployment YAML changed in the PR, it wouldn't be picked up unless it's called again and uses the YAML from the PR.

---

I see what you mean. I will take a look at it @dhaiducek

---

@dhaiducek  I found https://github.com/stolostron/governance-policy-addon-controller/blob/main/pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml managedclusteraddon has own deployment template. I guess it is better to change only images what you think?

---

However, propagator and addon controller, we don't need to skip kind-deploy-controller

---

I think I'd expect the addon-controller's YAML and the YAML here to be consistent. If it's possible, I'd personally prefer if the deployment selectors here were updated to match the addon-controller's so that this does work?

---

If you prefer to use addon-controller's deployment template (https://github.com/stolostron/governance-policy-addon-controller/blob/main/pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml) we can just simply skip kind-deploy-controller because hosted mode in governance-policy-framework already use addon-controller's. You can check here https://github.com/stolostron/governance-policy-framework/blob/0903452ed5de2b5d99eba286f4a16ef1007ed064/Makefile#L353